json_annotation/lib/src/checked_helpers.dart

```diff
@@ -104,18 +104,25 @@ class CheckedFromJsonException implements Exception {
         innerStack = null;
 
   CheckedFromJsonException._(
-    this.innerError,
+    Object innerError,
```

Comment:
See the comment on the linked issue, it looks like this can be `Object this.innerError`? Weird but seems to work.

Reply:
Will try!